src/constants.py

```diff
@@ -0,0 +1,27 @@
+OK = "OK"
```

Comment:
I think the addition of this file reduces readability. Please can you undo this change.